financial-services-accelerator/components/org.wso2.financial.services.accelerator.common/src/main/resources/findbugs-include.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+ ~ Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ ~
+ ~ WSO2 LLC. licenses this file to you under the Apache License,
+ ~ Version 2.0 (the "License"); you may not use this file except
+ ~ in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing,
+ ~ software distributed under the License is distributed on an
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied. See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
```

Comment:
add EOF line

Reply:
Fixed in [cc9c4b8](https://github.com/wso2/financial-services-accelerator/pull/107/commits/cc9c4b84b93e5b94a06fb11199bd42edaa1d9ed1)